lib/streamlit/DeltaGenerator.py

```diff
@@ -2361,6 +2361,10 @@ def date_input(self, element, label, value=None, key=None):
         value : datetime.date/datetime.datetime
             The value of this widget when it first renders. This will be
             cast to str internally. Defaults to today.
+        min_date : datetime.date/datetime.datetime
```

Comment:
Should be called `min_value` to match what we do for sliders. Same with `max_value`.

Also, `datetime.date/datetime.datetime` should be `datetime.date or datetime.datetime` since we use the NumPy doc style. This applies to other arguments in this function too.